dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -36,7 +36,7 @@
     USER_NAME_NULL(10004, "user name is null", "用户名不能为空"),
     HDFS_OPERATION_ERROR(10006, "hdfs operation error", "hdfs操作错误"),
     TASK_INSTANCE_NOT_FOUND(10008, "task instance not found", "任务实例不存在"),
-    TENANT_CODE_EXIST(10009, "tenant code {0} already exists", "租户编码[{0}]已存在"),
+    TENANT_CODE_EXIST(10009, "os tenant code {0} already exists", "操作系统租户[{0}]已存在"),
```

Comment:
It is best to modify all TENANT_CODE_EXIST to OS_TENANT_CODE_EXIST

Reply:
Got it.